samples/custom-bootstrap/README.md

```diff
@@ -1,18 +1,22 @@
 # Custom Envoy Bootstrap Configuration
 
-This sample creates a simple helloworld service that bootstraps the Envoy proxy with a custom configuration file.
+This sample creates a simple helloworld service that bootstraps the Envoy proxy. There are 2 ways to implement this:
+1. With a custom configuration file. 
+2. With envoy filter
 
 ## Starting the service
 
 First, we need to create a `ConfigMap` resource with our bootstrap configuration.
 
 ```bash
 kubectl apply -f custom-bootstrap.yaml
+kubectl apply -f custom-bootstrap-envoy-filter.yaml
```

Comment:
The way this is written, you mention there are two ways to do this, which are numbered. When you add the sections in the later paragraphs you just list both without numbering. This seems to imply in these sections one must do both.

Could we maybe have the first few paragraphs set up as 2 tabs, one for each of the 2 methods (from First, through just before  We can create..`/